### DISCUSSION TITLE
JAX support for TPU board's CPU

### DISCUSSION THREAD

**Discussion by bilal2vec** [jax-ml/jax#4870] [General] [answered]
Is JAX support for using the CPU built into the TPU board something that might happen in the future? Just out of curiosity, how was data loading handled for the large scale MLperf benchmarks? Is it currently like Pytorch-xla where you need a VM for each TPU board?

**Comment by bilal2vec** [comment, marked as answer]
given the awesome news from neurips today I think this has pretty much been answered :)

**Reply by hawkinsp** [reply]
Hopefully you'll forgive our silence up until now! We didn't want to spoil the surprise.
